contracts/periphery/oracle/tests/integration.rs

```diff
@@ -803,11 +822,53 @@ fn consult2() {
             contract_addr: astro_token_instance.clone(),
         },
     ];
-    create_pair(
+
+    let pair_info = create_pair(
         &mut router,
         owner.clone(),
         user.clone(),
         &factory_instance,
+        vec![
+            Asset {
+                info: asset_infos[0].clone(),
+                amount: Uint128::from(2000_u128),
+            },
+            Asset {
+                info: asset_infos[1].clone(),
+                amount: Uint128::from(2000_u128),
+            },
+        ],
+    );
+
+    // try to provide less then 1000
```

Comment:
Why do we place pair contract logic integration tests within oracle crate?
I suggest to move this test under pair crate.